protocol-units/bridge/service/tests/relayer.rs

```diff
@@ -0,0 +1,347 @@
+use bridge_service::types::Amount;
+use bridge_util::chains::bridge_contracts::BridgeContractError;
+use bridge_util::chains::bridge_contracts::BridgeContractResult;
+use bridge_util::chains::bridge_contracts::BridgeTransferCompletedDetails;
+use bridge_util::chains::bridge_contracts::BridgeTransferInitiatedDetails;
+use bridge_util::types::AddressError;
+use bridge_util::types::BridgeAddress;
+use bridge_util::types::Nonce;
+use bridge_util::BridgeContractEvent;
+use bridge_util::BridgeContractMonitoring;
+use bridge_util::BridgeRelayerContract;
+use bridge_util::BridgeTransferId;
+use futures::SinkExt;
+use futures::{
+	channel::mpsc::{UnboundedReceiver, UnboundedSender},
+	Stream, StreamExt,
+};
```

Comment:
Can we group the imports to avoid re importing the same crate?

Reply:
Personalty I prefer to have separated import, it's easier when you've to move or copy code for example. Reimporting is impossible because of the compiler.
The last my tooling doesn't do it automatically.
There's a rule that we must group import?